src/wayland/seat/pointer.rs

```diff
@@ -219,6 +228,22 @@ impl PointerHandle {
             grab.axis(&mut handle, details);
         });
     }
+
+    /// Access the current location of this pointer in the global space
+    pub fn current_location(&self) -> (f64, f64) {
+        self.inner.borrow().location
+    }
+}
+
+/// Data about the event that started the grab.
+#[derive(Clone)]
+pub struct GrabStartData {
+    /// The focused surface and its location, if any, at the start of the grab.
```

Comment:
I think this is the location of the pointer relative to the surface, not the location of the surface itself. This module has no knowledge of the surface locations.

Reply:
This location comes from the motion handler, which (in anvil) gets it from `WindowMap::get_surface_under`, which returns the surface location. But, as I said in the IRC, I don't actually see this field used for any purpose anywhere, so it might as well be not what was intended.

Reply:
Ah right, it's been too long I touched at this part, sorry.